timelock-api/src/main/conjure/timelock-api.yml

```diff
@@ -20,8 +20,12 @@ types:
       base-type: any
       external:
         java: com.palantir.lock.v2.Lease
+    LockWatchStateUpdate:
```

Comment:
I think I missed this earlier, any reason to *not* use Conjure unions here? you should still get the visitors and everything that you want etc.

Reply:
Discussed offline -- requires nontrivial refactoring since the update is used outside of timelock api